src/basic/path-util.c

```diff
@@ -1441,3 +1441,36 @@ int path_glob_can_match(const char *pattern, const char *prefix, char **ret) {
                 *ret = NULL;
         return false;
 }
+
+const char* default_path(void) {
```

Comment:
can we maybe call this default_exec_path() or so? or default_search_path() or default_binary_search_path() or something?

"default_path" is a bit too generic a function for me.

Reply:
i mean, when it was an uppercase macro then it at least hat visual resemblence with the shell variable, but in lowercase it just sounds way too generic.